stylesheets/_mixin.responsive-utilities.scss

```diff
@@ -0,0 +1,90 @@
+/**
+ * Shortcut for easy hiding/showing at different breakpoints
+**/
+
+@mixin sr-hide {
+    border: 0 !important;
+    clip: rect(0 0 0 0);
+    height: 1px;
+    margin: -1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    width: 1px;
+}
+
+@mixin hide {
+    display: none !important;
+}
+
+@mixin visible-on-phone {
+    @include respond-min($screen-phone) {
+        @include hide;
+    }
+}
+
+@mixin sr-visible-on-phone {
+    @include respond-min($screen-phone) {
+        @include sr-hide;
+    }
+}
+
+@mixin visible-on-tablet {
+    @media not all and (min-width: $screen-tablet) and (max-width: $screen-desktop - 1) {
+        @include hide;
+    }
+}
+
+@mixin sr-visible-on-tablet {
+    @media not all and (min-width: $screen-tablet) and (max-width: $screen-desktop - 1) {
```

Comment:
Ignore these, just realised it's a hack to get it to do the reverse logic. Could we write a new mixin for this, or amend the `respond-min-max` one to include a check that adds this for you?